app/src/main/res/values-fr/strings.xml

```diff
@@ -234,6 +241,11 @@
   <string name="miband_prefs_reserve_alarm_calendar">Alarmes à réserver pour événements futurs</string>
   <string name="miband_prefs_hr_sleep_detection">Utiliser le capteur cardiaque pour améliorer la précision du sommeil</string>
   <string name="miband_prefs_device_time_offset_hours">La compensation de temps en heure (pour détecter le sommeil de travailleurs en rotation, par exemple)</string>
+  <string name="miband2_prefs_dateformat">Mi band 2 : format de l\'heure</string>
+  <string name="dateformat_time">Heure seule</string>
+  <string name="dateformat_date_time"><![CDATA[Heure & date]]></string>
+  <string name="mi2_prefs_activate_display_on_lift">Alumer quand le poignet se lève</string>
```

Comment:
Yes, of course, how should I correct this in the pull request? Do I change in my copy and do the pull request again or can you correct after the merge?